src/briefcase/platforms/web/static.py

```diff
@@ -455,14 +456,51 @@ class StaticWebPublishCommand(StaticWebMixin, PublishCommand):
 class StaticWebDevCommand(StaticWebMixin, DevCommand):
     description = "Run a static web project in development mode. (Work in progress)"
 
-    def run_dev_app(self, app: AppConfig, env, passthrough=None, **kwargs):
-        raise UnsupportedCommandError(
-            platform="web",
-            output_format="static",
-            command="dev",
-        )
+    def __call__(
+        self,
+        appname=None,
+        update_requirements=False,
+        run_app=True,
+        test_mode=False,
+        passthrough=None,
+        **options,
+    ):
+        if len(self.apps) == 1:
+            app = list(self.apps.values())[0]
+        elif appname:
+            try:
+                app = self.apps[appname]
+            except KeyError as e:
+                raise BriefcaseCommandError(
+                    f"Project doesn't define an application named '{appname}'"
+                ) from e
 
-    # implement logic to run the web server in development mode
+        else:
+            raise BriefcaseCommandError(
+                "Project specifies more than one application; use --app to specify which one to start."
+            )
+        # Confirm host compatibility, that all required tools are available,
+        # and that the app configuration is finalized.
+        self.finalize(app, test_mode)
+
+        self.verify_app(app)
+
+        with virtual_environment(
+            tools=self.tools,
+            console=self.console,
+            base_path=self.base_path,
+            app=app,
+            **options,
+        ) as venv_path:
+            self.console.info(f"Virtual environment created at: {venv_path}")
```

Comment:
If the virtual environment context manager is generating output, there's no need to output it again.